react-ui-kit/src/Inputs/Suggest/components/Input.tsx

```diff
@@ -7,24 +7,24 @@ import { Input as BaseInput } from '../../BaseField';
 import { StyledSearchOutlineIcon } from '../StyledSearchOutlineIcon';
 import { ReactComponent as SearchOutlineIcon } from '../../../Icons/system/SearchOutlineIcon.svg';
 
-export const Input: any = ({ selectProps: { inputProps }, onChange, onBlur, onFocus }: any) => {
-  const { disabled, value, size, status, clearable, setValueInput } = inputProps;
+import { IconWrapper } from './style';
+
+export const Input: any = ({ selectProps: { inputProps }, onChange, onBlur, onFocus }) => {
+  const { disabled, value, size = 'big', status, clearable, setValueInput } = inputProps;
   const [errorActive, setErrorActive] = useState<boolean>(false);
 
   const handleChange = (event: ChangeEvent<HTMLInputElement>, value: string) => {
     onChange(event, value);
     setValueInput(value);
-    if (inputProps.onChange) {
-      inputProps.onChange(event, value);
-    }
+
+    inputProps.onChange(event, value);
   };
 
   const handleBlur = (event: FocusEvent<HTMLInputElement>, value: string) => {
     onBlur(event);
     setErrorActive(false);
-    if (inputProps.onBlur) {
-      inputProps.onBlur(event, value);
-    }
+
+    inputProps.onBlur && inputProps.onBlur(event, value);
```

Comment:
Можно ещё короче записать :)
```suggestion
    inputProps.onBlur?.(event, value);
```
Но я не настаиваю